new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java

```diff
@@ -77,4 +82,48 @@ public static boolean patchDomainCustomResource(String domainUid, String namespa
       String patchFormat) {
     return Kubernetes.patchDomainCustomResource(domainUid, namespace, patch, patchFormat);
   }
+
+  /**
+   * Scale the cluster of the domain in the specified namespace.
+   *
+   * @param domainUid domainUid of the domain to be scaled
+   * @param namespace namespace in which the domain resides
+   * @param clusterName name of the WebLogic cluster to be scaled in the domain
+   * @param numOfServers number of servers to be scaled to
+   * @return true if success, false otherwise
```

Comment:
what is success? you submitted the async request?   it worked? something else?

Reply:
fixed